src/Aspire.Dashboard/Model/BrowserTimeProvider.cs

```diff
@@ -29,11 +29,11 @@ public void SetBrowserTimeZone(string timeZone)
     {
         if (!TimeZoneInfo.TryFindSystemTimeZoneById(timeZone, out var timeZoneInfo))
```

Comment:
Does this work cross plat?

Reply:
I believe so. They're standardized timezone ids. I don't have Linux/macos to test but no one has reported this as a problem.